### ISSUE TITLE
How about the result with SSD?

### ISSUE THREAD

**Issue by weiweili123** [BOBrown/SSD-Centerloss#1]
I'm glad to see your work with centerloss,have you got some better performance with center loss than ohem in ssd?

**Comment by BOBrown** [comment]
@weiweili123 Yes, I trained SSD with center loss on our own datasets, similar to the MRI scanning image. The performance is better than the SSD.
